apps/api/src/ormconfig.ts

```diff
@@ -1,25 +1,25 @@
 import * as dotenv from 'dotenv';
+import { DataSource } from 'typeorm';
 import { PostgresConnectionOptions } from 'typeorm/driver/postgres/PostgresConnectionOptions';
 import { DatabaseNamingStrategy } from './database/database.naming-strategy';
+
 dotenv.config();
 // Check typeORM documentation for more information.
 
-const config: PostgresConnectionOptions = {
+export const config: PostgresConnectionOptions = {
   host: process.env.POSTGRES_HOST,
   type: 'postgres',
-  port: 5432,
+  port: +(process.env.PORTGRES_PORT || 5432),
   connectTimeoutMS: 5000,
-  username: process.env.POSTGRES_USERNAME,
+  username: process.env.POSTGRES_USERNAME || 'freshworks',
   password: process.env.POSTGRES_PASSWORD,
-  database: process.env.POSTGRES_DATABASE,
-  cli: {
-    migrationsDir: 'src/migration',
-    entitiesDir: 'src/**/entity/*.entity.ts',
-  },
+  database: process.env.POSTGRES_DATABASE || 'ehrp',
+  entities: ['dist/**/*.entity.js'],
+  migrations: ['dist/migration/*.js'],
```

Comment:
Whats the reason to pull from dist instead of src?

Reply:
I mainly copied what Kushal did for TBCM. I saw a few examples of people having issues in production. Since this never had that issue I just changed it back.